docs/src/guides/organizing-stores.mdx

```diff
@@ -260,7 +260,7 @@ Example(
 ...
 ```
 
-A side effect of this is that writing widget tests become easier as well as we'd know what dependencies a widget has. We could then pass, say, mocks (e.g. using the [mockito][2] package] of said dependencies can be passed if required.
+A side effect of this is that writing widget tests become easier as well. Since the widget's dependencies have been made explicit, we could pass mocks (e.g. using the [mockito][2] package) of said dependencies if required.
```

Comment:
a comma is needed after "said dependencies"

Reply:
It's not said with a pause to warrant a comma. A similar usage would be, say, "you can call me if needed"

Reply:
got it